src/smt/term_formula_removal.cpp

```diff
@@ -81,6 +86,13 @@ theory::TrustNode RemoveTermFormulas::run(
   return theory::TrustNode::mkTrustRewrite(assertion, itesRemoved, d_tpg.get());
 }
 
+theory::TrustNode RemoveTermFormulas::run(Node assertion)
```

Comment:
```suggestion
theory::TrustNode RemoveTermFormulas::run(TNode assertion)
```
The same in the header, and for the other overload of `run()` as well.